RadixWallet/Features/AssetTransferFeature/Components/ChooseAccount/ChooseReceivingAccount+Reducer.swift

```diff
@@ -22,7 +24,7 @@ public struct ChooseReceivingAccount: Sendable, FeatureReducer {
 			case invalid
 		}
 
-		func validateManualAccountAddress() -> AddressValidation {
+		var validatedManualAccountAddress: AddressValidation {
```

Comment:
Minor: it is quite confusing to check medium-big states like this with stored and computed properties all mixed. I believe it could help to just have the stored properties on the `State` definition, and then create a `extension ChooseReceivingAccount.State` where we define the `enum AddressValidation` and the two computed properties (`validatedManualAccountAddress` & `validatedAccountAddress`).

Reply:
Agree